pyproject.toml

```diff
@@ -16,7 +16,8 @@ dependencies = [
     "accelerate", "fastapi", "gradio==3.23", "markdown2[all]", "numpy",
     "prompt_toolkit>=3.0.0", "requests", "rich>=10.0.0", "sentencepiece",
     "shortuuid", "transformers>=4.28.0", "tokenizers>=0.12.1", "torch",
-    "uvicorn", "wandb", "httpx", "shortuuid", "pydantic",
+    "uvicorn", "wandb", "httpx", "shortuuid", "pydantic", "black==23.3.0",
+    "pylint==2.8.2",
```

Comment:
How about this https://peps.python.org/pep-0631/#optional-dependencies?

Reply:
good idea. fixed